modules/distribution/src/main/scripts/fluo

```diff
@@ -170,7 +166,7 @@ init)
       print_usage
     fi
     if [[ "$app" = *"-h"* ]]; then
-      java org.apache.fluo.command.FluoInit "$FLUO_CONN_PROPS" none /dev/null "$2"
+      $JAVA org.apache.fluo.command.FluoInit "$FLUO_CONN_PROPS" none /dev/null "$2"
```

Comment:
this could be a follow on issue, but I am wondering if all commands should be able to use the -D args.